src/components/shared/Profile.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+// Libaries
+import styled from 'styled-components';
+
+import { Para1, Para2 } from '../index';
+
+export const limitString = (str, n) =>
+  // eslint-disable-next-line
+  str ? (str.length >= n ? `${str.substr(0, n)}...` : str) : null;
+
+const Designation = styled(Para2)`
+  font-style: italic;
+  text-align: center;
+  color: ${({ dark }) => dark ? `#ffffff` : `#000000`};
+`;
+
+const ImageContainer = styled.div`
+    width: ${({ small }) => (small ? `110px` : `180px`)};
+    height: ${({ small }) => (small ? `110px` : `180px`)};
+    margin: 0px 5px;
+    margin-top: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+`;
+
+const Img = styled.img`
+    width: ${({ small }) => (small ? `90px` : `120px`)};
+    height: ${({ small }) => (small ? `90px` : `120px`)};
+    border-radius: 100%;
+    background: var(--color-secondary);
```

Comment:
please add the following properties so as to maintain appropriate aspect ratios
```
object-fit: cover;
object-position: center;
```

Reply:
ok